user/preprocess/input.md

```diff
@@ -472,15 +473,16 @@ condition is:
             {
                 "file" : "friction-constraints.txt"
             }
-        ],
-    },
-    "external_loading_conditions": {
-        "concentrated_nodal_forces": [
-          {
-               "file": "nodal-tractions.txt"
-          }
         ]
     }
+  },
+  "external_loading_conditions": {
```

Comment:
Could you please remove this `external_loading_conditions` from here because this subsection is about boundary conditions and move it to loading? as an additional example of how to call loading from a file?

Reply:
https://mpm.cb-geo.com/#/user/preprocess/input?id=loading Please add the external_loading_condition in this section after the current example

Reply:
Done!